src/controller/articlesController.js

```diff
@@ -305,6 +297,18 @@ class ArticleController {
           article.viewStats,
           article.title
         );
+        const getTags = await ArticleTag.findAll({
+          where: {
+            articleId: article.id
+          },
+          include: [{
+            model: Tags,
+          }]
+        });
+        const getTagNames = getTags
+          .map(item => item.dataValues.Tag.dataValues.tagName);
```

Comment:
Why are we doing this? Is there no other way to get tag names without looping and chaining properties this deep?

Reply:
* We tried to include the Tags model when fetching an article but it was returning an empty array. The relation between article and tag is many-to-many.
* So, we fetched the tags for an article separately which returns an array of objects.